build.gradle

```diff
@@ -102,10 +104,11 @@ subprojects {
         junit_engine = "org.junit.jupiter:junit-jupiter-engine:$junitVersion"
         junit_params = "org.junit.jupiter:junit-jupiter-params:$junitVersion"
         llzip = "software.coley:lljzip:$lljzipVersion"
-        logback_classic = 'ch.qos.logback:logback-classic:1.2.10' // newer releases break in jar releases
-        pe = 'com.github.cademtz:JavaPeParser:1.0.0' // fails on 2.2.3, breaking changes
+        slf4j = "org.slf4j:slf4j-api:$slf4jVersion"
+        logback_classic = "ch.qos.logback:logback-classic:$logbackVersion"
```

Comment:
The comment for logback-classic still applies and should not be updated.

Running `gradlew build` and attempting to run the created jar fails:
```
SLF4J: A SLF4J service provider failed to instantiate:
org.slf4j.spi.SLF4JServiceProvider: Provider ch.qos.logback.classic.spi.LogbackServiceProvider not found
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
Exception in thread "main" java.lang.ClassCastException: class org.slf4j.helpers.NOPLoggerFactory cannot be cast to class ch.qos.logback.classic.LoggerContext (org.slf4j.helpers.NOPLoggerFactory and ch.qos.logback.classic.LoggerContext are in unnamed module of loader 'app')
        at me.coley.recaf.util.logging.Logging.addFileAppender(Logging.java:83)
        at me.coley.recaf.Main.setupLogging(Main.java:85)
        at me.coley.recaf.Main.main(Main.java:39)
        ```

Reply:
![image](https://github.com/Col-E/Recaf/assets/39942874/60247fb3-5c8a-4c14-83af-82eda5acad6b)
This problem should be caused by the mismatch between slf4j and logback versions. This problem no longer exists. You can clone my code and try it.
```bash
git clone -b dev3-optimize --single-branch https://github.com/meiMingle/Recaf.git Recaf
cd Recaf
gradle build
```

Reply:
Now it's fixed